web/memplots.js

```diff
@@ -281,6 +292,12 @@ function drawHistograms(rootUrl) {
   );
 }
 
+function drawPlots(rootUrl) {
+  tooltip = $('#tooltip');
```

Comment:
There are outstanding calls to `$('#tooltip')`. Perhaps we could replace them by the global var too?